02week/blog/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>The Best Blog On Earth</title>
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+    <header>
+      <h1>The Best Blog On Earth</h1>
+    </header>
+      <main>
+        <body>
+          <section class="mainbox">
+          <article class="article1">
```

Comment:
It's good for a class example like this to have classes like 'article1', 'article2', etc. if you want to style them differently, but in a real project, you would just have one like this: class="blogArticle"